lib/java/com/google/android/material/textview/MaterialTextView.java

```diff
@@ -88,7 +90,9 @@ public MaterialTextView(
 
   public MaterialTextView(
       @NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
-    super(context, attrs, defStyleAttr);
+    super(wrap(context, attrs, defStyleAttr, 0), attrs, defStyleAttr);
```

Comment:
```suggestion
    super(wrap(context, attrs, defStyleAttr, defStyleRes), attrs, defStyleAttr);
```

Reply:
Good catch, I went into copy-paste mode 🙈

Do you know why TextView is one of the few widgets that supports this constructor?